docs/notebooks/README.md

```diff
@@ -22,6 +22,7 @@ on your system.
 1. Copy the text below to a file named `viz.txt`.
 
 ```
+bqplot
```

Comment:
I think this needs to be added here as well - https://github.com/heremaps/xyz-spaces-python/blob/master/binder/environment.yml for the binder.

Reply:
Yep, good catch!